ledger/ledger-api-bench-tool/src/main/scala/com/daml/ledger/api/benchtool/submission/RandomnessProvider.scala

```diff
@@ -4,7 +4,7 @@
 package com.daml.ledger.api.benchtool.submission
 
 trait RandomnessProvider {
-  def randomDouble(): Double // 0.0 <= randomDouble() <= 1.0
+  def randomDouble(): Double // 0.0 <= randomDouble() < 1.0
```

Comment:
For `scala.util.Random.nextDouble`, the upper bound is exclusive.

Reply:
Thanks for spotting this! I need to review `Distribution` implementation and tests to check if this is actually reflected there.